### PR TITLE
fix auto approval flag

### PR DESCRIPTION
**Pull request by skeeey**

### CODE REVIEW

File: pkg/hub/manager.go [hunk 1]
```diff
@@ -51,8 +51,8 @@ func NewHubManagerOptions() *HubManagerOptions {
 // AddFlags registers flags for manager
 func (m *HubManagerOptions) AddFlags(fs *pflag.FlagSet) {
 	features.DefaultHubMutableFeatureGate.AddFlag(fs)
-	fs.StringArrayVar(&m.ClusterAutoApprovalUsers, "--cluster-auto-approval-users", m.ClusterAutoApprovalUsers,
-		"A list of reachable spoke cluster api server URLs for hub cluster.")
+	fs.StringArrayVar(&m.ClusterAutoApprovalUsers, "cluster-auto-approval-users", m.ClusterAutoApprovalUsers,
+		"A bootstrap user list whoes cluster registration requests can be automatically approved.")
```
Comment (morvencao) [inline pkg/hub/manager.go]: s/whoes/whose/ ?

Comment (skeeey) [reply pkg/hub/manager.go]: correct

**Review by morvencao — COMMENTED** [review]
/lgtm

**Pull request merged**